main.js

```diff
@@ -1,6 +1,31 @@
 require('es6-promise').polyfill();
 var rest = require('restler');
 
+// Object.assign polyfill
+if (typeof Object.assign != 'function') {
```

Comment:
Thanks! This is only the 3rd PR in my life. Your comments are priceless!
Am I supposed apply changes you suggested and push again?